### PR TITLE
feat: intro update

### PR DESCRIPTION
**Pull request by nissimsan**
Intro suggestion, ref https://github.com/w3c-ccg/traceability-vocab/issues/719. 
Input and suggestions very welcome!

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -204,12 +204,41 @@
       <h2>Introduction</h2>
 
       <p>
-        This specification is designed to enable global supply chain stakeholders to benefit from emerging technological web
-        standards like the Verifiable Credential (VC) Data Model, Linked Data (i.e. JSON-LD), and the Decentralized Identifier
-        (DID) Spec. This specification is relevant to prevalent supply chains but also allows for other use cases across a
-        variety of market segments. The corresponding data exchange schemas are modeled for Verifiable Credentials, which is a
-        standard for cryptographically signing business data. Schema semantics are precisely defined by use of Linked Data
-        which leverage established vocabularies such as schema.org, GS1, and UN/CEFACT.
+        This specification seeks to enable a path towards supply chain digitization, at its foundation addressing new threats surfaces
```
Comment (TallTed) [inline docs/index.html:207]:
```suggestion
        This specification seeks to enable a path towards supply chain digitization, at its foundation addressing new threat surfaces 
```

File: docs/index.html [hunk 2]
```diff
@@ -204,12 +204,41 @@
...
+        This specification seeks to enable a path towards supply chain digitization, at its foundation addressing new threats surfaces 
+        of a digital society:
+      </p>
+
+      <p>
+        <cite>
+          The private sector must adapt to the continuously changing threat environment, ensure its products are built and operate 
+          securely, and partner with the Federal Government to foster a more secure cyberspace.  In the end, the trust we place in 
+          our digital infrastructure should be proportional to how trustworthy and transparent that infrastructure is, and to the 
+          consequences we will incur if that trust is misplaced.
```
Comment (TallTed) [inline docs/index.html:216]:
```suggestion
          our digital infrastructure should be proportional to [the trustworthiness and transparency of that infrastructure, and] to the 
          consequences [that may be incurred] if that trust is misplaced.
```

File: docs/index.html [hunk 3]
```diff
@@ -204,12 +204,41 @@
...
+        <br/>
+        Source: <a href="https://www.whitehouse.gov/briefing-room/presidential-actions/2021/05/12/executive-order-on-improving-the-nations-cybersecurity/">
+          https://www.whitehouse.gov/briefing-room/presidential-actions/2021/05/12/executive-order-on-improving-the-nations-cybersecurity/</a>
+      </p>
+      
+      <p>
+        <cite>
+          The digital transformation of society [...] has expanded the threat landscape and is 
+          bringing about new challenges, which require adapted and innovative responses.
+          Now any disruption, even one initially confined to one entity or one sector, can have cascading effects more broadly,
```
Comment (TallTed) [inline docs/index.html:227]:
```suggestion
          Now any disruption, even one initially confined to one entity or one sector, can have [more broad, cascading effects], 
```

File: docs/index.html [hunk 4]
```diff
@@ -204,12 +204,41 @@
...
+          internal market.
+        </cite>
+        <br/>
+        Source: <a href="https://digital-strategy.ec.europa.eu/en/library/proposal-directive-measures-high-common-level-cybersecurity-across-union">
+          https://digital-strategy.ec.europa.eu/en/library/proposal-directive-measures-high-common-level-cybersecurity-across-union</a>
+      </p>
+      <p>
+        In doing so, this specification is designed to enable global supply chain stakeholders to benefit from emerging web 
+        technology standards like the Verifiable Credential (VC) Data Model, Linked Data (JSON-LD), and the Decentralized 
+        Identifier (DID) Spec. This specification is relevant to prevalent supply chains but also allows for other use cases
```
Comment (TallTed) [inline docs/index.html:238]:
```suggestion
        technology standards like the Verifiable Credential Data Model (VCDM), JSON for Linked Data (JSON-LD), and Decentralized 
        Identifiers (DIDs). This specification is relevant to prevalent supply chains, and also allows for other use cases 
```

File: docs/index.html [hunk 5]
```diff
@@ -204,12 +204,41 @@
...
+        Source: <a href="https://digital-strategy.ec.europa.eu/en/library/proposal-directive-measures-high-common-level-cybersecurity-across-union">
+          https://digital-strategy.ec.europa.eu/en/library/proposal-directive-measures-high-common-level-cybersecurity-across-union</a>
+      </p>
+      <p>
+        In doing so, this specification is designed to enable global supply chain stakeholders to benefit from emerging web 
+        technology standards like the Verifiable Credential (VC) Data Model, Linked Data (JSON-LD), and the Decentralized 
+        Identifier (DID) Spec. This specification is relevant to prevalent supply chains but also allows for other use cases 
+        across a variety of market segments. The corresponding data exchange schemas are modeled for Verifiable Credentials, 
+        which is a standard for cryptographically signing business data. Schema semantics are precisely defined by use of Linked 
+        Data which leverage established web vocabularies.
```
Comment (TallTed) [inline docs/index.html:241]:
```suggestion
        which is a standard for cryptographically signing business data. Schema semantics are precisely
        defined by use of JSON for Linked Data which leverages established web vocabularies.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Editorial...

File: docs/index.html [hunk 6]
```diff
@@ -204,12 +204,41 @@
...
-        standard for cryptographically signing business data. Schema semantics are precisely defined by use of Linked Data
-        which leverage established vocabularies such as schema.org, GS1, and UN/CEFACT.
+        This specification seeks to enable a path towards supply chain digitization, at its foundation addressing new threats surfaces 
+        of a digital society:
+      </p>
+
+      <p>
+        <cite>
+          The private sector must adapt to the continuously changing threat environment, ensure its products are built and operate 
+          securely, and partner with the Federal Government to foster a more secure cyberspace.  In the end, the trust we place in
```
Comment (mkhraisha) [inline docs/index.html:214]: Should still maintain our international focus, should not refer to "the federal government" or similar, should be framed in working with governments frame

**Review by mkhraisha — CHANGES_REQUESTED** [review]

File: docs/index.html (hunk 6 above)
Comment (TallTed) [reply docs/index.html:214]:
```suggestion
          securely, and partner with governments to foster a more secure cyberspace.  In the end, the trust we place in 
```

File: docs/index.html [hunk 7]
```diff
@@ -204,8 +204,10 @@
       <h2>Introduction</h2>
 
       <p>
-        This specification seeks to enable a path towards supply chain digitization, at its foundation addressing new threats surfaces 
-        of a digital society:
+        In streamlining operations and reacting on increasing market transparency and traceability expectations, 
+        global supply chains are continuously undergoing digital transformation. At the same pace, the exposed 
+        surfaces of cyber attacks are increasing. These vunerabilities to critical society infrastructure are
```
Comment (TallTed) [inline docs/index.html:209]:
```suggestion
        In streamlining operations and reacting to increasing market transparency and traceability expectations, 
        global supply chains are continuously undergoing digital transformation. At the same time, the surfaces
        exposed to cyber attacks are increasing. These vulnerabilities of critical society infrastructure are 
```

File: docs/index.html [hunk 8]
```diff
@@ -220,6 +222,10 @@ <h2>Introduction</h2>
           https://www.whitehouse.gov/briefing-room/presidential-actions/2021/05/12/executive-order-on-improving-the-nations-cybersecurity/</a>
       </p>
       
+      <p>
+        As well as in this Digital Strategy statement from the European Commission:
```
Comment (TallTed) [inline docs/index.html:226]:
```suggestion
        Similar recognition is seen in this Digital Strategy statement from the European Commission: 
```

File: docs/index.html [hunk 9]
```diff
@@ -233,12 +239,9 @@ <h2>Introduction</h2>
...
       <p>
-        In doing so, this specification is designed to enable global supply chain stakeholders to benefit from emerging web 
-        technology standards like the Verifiable Credential (VC) Data Model, Linked Data (JSON-LD), and the Decentralized 
-        Identifier (DID) Spec. This specification is relevant to prevalent supply chains but also allows for other use cases 
-        across a variety of market segments. The corresponding data exchange schemas are modeled for Verifiable Credentials, 
-        which is a standard for cryptographically signing business data. Schema semantics are precisely defined by use of Linked 
-        Data which leverage established web vocabularies.
+        The goal of this specification is to provide a path towards a secure digitized global supply chain. It does so by 
+        leveraging modern cryptography and web technology standards like the Verifiable Credential (VC) Data Model, Linked Data 
+        (JSON-LD), and the Decentralized Identifier (DID) Specification.
```
Comment (TallTed) [inline docs/index.html:244]:
```suggestion
        leveraging modern cryptography and web technology standards like the <a 
        href="https://www.w3.org/TR/vc-data-model-2.0/">Verifiable Credential Data Model</a> (<a 
        href="https://www.w3.org/TR/vc-data-model-2.0/">VCDM</a>), <a 
        href="https://www.w3.org/TR/json-ld11/">JSON for Linked Data</a> (<a 
        href="https://www.w3.org/TR/json-ld11/">JSON-LD</a>), and <a 
        href="https://www.w3.org/TR/did-core/">Decentralized Identifiers</a> (<a 
        href="https://www.w3.org/TR/did-core/">DIDs</a>).
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Review by mkhraisha — APPROVED** [review]
Looks great!

File: docs/index.html [hunk 10]
```diff
@@ -204,27 +204,33 @@
       <h2>Introduction</h2>
 
       <p>
-        This specification seeks to enable a path towards supply chain digitization, at its foundation addressing new threats surfaces 
-        of a digital society:
+        In streamlining operations and reacting to increasing market transparency and traceability expectations, 
+        global supply chains are continuously undergoing digital transformation. At the same time, the surfaces
```
Comment (TallTed) [inline docs/index.html:208]:
```suggestion
        In streamlining operations and reacting to increasing expectations of market transparency and traceability,
        global supply chains are undergoing continuous digital transformation. At the same time, the surfaces
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Review by brownoxford — APPROVED** [review]

**Review by TallTed — APPROVED** [review]

**Pull request merged**